capsules/segmenter_person_openvino/licenses/model.LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
We're not consistent about this, but the license should include the copyright owner as well, which for this model is "Copyright (c) 2019 Intel Corporation".